### PR TITLE
MP-40: Adding GTM tracking

### PR DESCRIPTION
**Pull request by alejandronanez**

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -1,12 +1,11 @@
 <head>
   <!-- Global site tag (gtag.js) - Google Analytics -->
-  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-27002928-1"></script>
+  <script async src="https://www.googletagmanager.com/gtm.js?id=GTM-K7GFJTV"></script>
```
Comment (alejandronanez) [inline _includes/head.html]: Updating this to use the right tracking using Google Tag Manager

Slack convo: https://zapier.slack.com/archives/C028W5V2A/p1559853829094900?thread_ts=1559851688.091600&cid=C028W5V2A

File: _includes/javascript-docs-tracking.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          message: 'navigate'
+        }),
+      });
+    }
+
+    document.addEventListener('DOMContentLoaded', function() {
+      sendToGa();
```
Comment (alejandronanez) [inline _includes/javascript-docs-tracking.html]: Sending tracking information once the page is loaded.

Comment (kangzeroo) [reply _includes/javascript-docs-tracking.html]: This is the `prod` GTM code. How can we test this?

**Comment by alejandronanez** [comment]
@kangzeroo there's no easy way to test it, we should test it once it goes live.

FWIW, we had the wrong code for a long time, so, there's nothing we can actually break.

**Pull request merged**